src/main/java/com/softserve/actent/model/entity/Event.java

```diff
@@ -0,0 +1,124 @@
+package com.softserve.actent.model.entity;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import com.softserve.actent.constant.StringConstants;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.validator.constraints.Length;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.ForeignKey;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.Set;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "events")
+public class Event {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NonNull
+    @NotBlank(message = StringConstants.TITLE_SHOULD_NOT_BE_BLANK)
+    @Length(max = 100, message = "Too long")
+    @Column(nullable = false, length = 100)
+    private String title;
+
+    @Length(max = 500, message = "Too long")
+    @Column(length = 500)
+    private String description;
+
+    @CreationTimestamp
+    @Column(name = "creation_date")
+    private Timestamp creationDate;
+
+    @NonNull
+    @Column(name = "start_date", nullable = false)
+    private Long startDate;
+
+    @NonNull
+    @Column(nullable = false)
+    private Long duration;
+
+    @NonNull
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
```

Comment:
Remove FetchType parameter, as it goes EAGER by default, which is preferred for us in -to-one cases.

Reply:
Do not use EAGER! Ask Igor!

Reply:
Why don't use it for fetching single fields, not whole collections? 
@Pitcity whats wrong with this?

Reply:
For example — I have Student entity, which have University field. When I'm getting particular student, I want to get info about university to which he belongs and not additionally ask for it, so that's EAGER fetching.
From the other side, I won't want to have whole set of students, when I'll get particular university, and this will be LAZY fetching.

Reply:
Ask Igor